### PR TITLE
Feat: wishlist

### PR DESCRIPTION
**Pull request by thesudhansupanda**

### CODE REVIEW

File: wishlist.html [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+            <div class="wishActionButtons pd-r-sm">
+                <button class="btn btn-success mg-y-sm"><i class="fas fa-shopping-cart mg-r-sm"></i>Add to
+                    Cart</button>
+            </div>
+        </div>
+    </div>
+    
+    
+ </div>
+
```
Comment (shashidwivedi) [inline wishlist.html:127]: extra lines present

File: styles/wishlist.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.wishlisttst {
+
```
Comment (shashidwivedi) [inline styles/wishlist.css:2]: extra lines present. You can use prettier

**Review by shashidwivedi — APPROVED** [review]
Rest looks good. You can merge

File: styles/wishlist.css (hunk 2 above)
Comment (thesudhansupanda) [reply styles/wishlist.css:2]: alright, will do as part of refactor

**Review by krishanu2911 — APPROVED** [review]
No changes are Required.

**Pull request merged**